diff --git "a/C:/Users/satre/Downloads/extensionsModern/vscode-lua-master/client\\src\\languageserver.ts" "b/C:/Users/satre/Downloads/extensionsModern/faf-lua-vscode-extension-patch-main/client\\src\\languageserver.ts"
index dc91729..5f04c26 100644
--- "a/C:/Users/satre/Downloads/extensionsModern/vscode-lua-master/client\\src\\languageserver.ts"
+++ "b/C:/Users/satre/Downloads/extensionsModern/faf-lua-vscode-extension-patch-main/client\\src\\languageserver.ts"
@@ -224,7 +224,6 @@ class LuaClient extends Disposable {
     this.statusBar();
     this.languageConfiguration();
     this.provideHover();
-    this.dataExport();
   }
 
   private async getCommand(config: vscode.WorkspaceConfiguration) {
@@ -339,15 +338,6 @@ class LuaClient extends Disposable {
     this.disposables.push(bar);
   }
 
-  dataExport() {
-    this.disposables.push(
-      Commands.registerCommand("FALua.ExportData", async () => {
-        const result = await this.client.sendRequest("$/export/data");
-        console.log("##FA", result);
-      }),
-    );
-  }
-
   private onCommand() {
     if (!this.client) {
       return;
